src/provider/FDC3.ts

```diff
@@ -254,7 +254,7 @@ export class FDC3 {
                             clearTimeout(timeout);
                             if (context) {
                                 // Pass context to application before resolving
-                                fin.InterApplicationBus.publish('context', {context}).then(resolve).catch((reason: string) => reject(new Error(reason)));
+                                this.apiHandler.channel.dispatch(app.identity, 'context', {context});
```

Comment:
Just want to check we're happy to lose the error handling here

Reply:
Not just the error handling, but the `.then(resolve)` was also removed. This is what makes sure we have sent the context before resolving the `open` call, and should definitely be there.